MDANSE_GUI/Src/MDANSE_GUI/MolecularViewer/MolecularViewer.py

```diff
@@ -268,8 +281,136 @@ def _new_trajectory_object(self, fname: str, trajectory: Trajectory):
         data : Trajectory
             instance of the MDANSE input trajectory handler
         """
+
         reader = hdf5wrapper.HDF5Wrapper(fname, trajectory, trajectory.chemical_system)
         self.set_reader(reader)
+        self._animation_timer.stop()
+
+    def create_actor_for_placeholder_3d_model(
+        self,
+        filename: str,
+        colour: tuple[float, float, float],
+        scale: float,
+        position: tuple[float, float, float],
+    ):
+        """_summary_
+
+        Parameters
+        ----------
+        filename : str
+            _description_
+        colour : tuple[float, float, float]
+            _description_
+        scale : float
+            _description_
+        position : tuple[float, float, float]
+            _description_
+
+        Returns
+        -------
+        vtk.vtkActor
+            The created actor
+        """
+        # Validate colour
+        if not isinstance(colour, tuple):
+            raise TypeError("Colour must be a tuple.")
+        if len(colour) != 3:
+            raise ValueError("Colour must be a tuple of three values.")
+        if not all(isinstance(x, float) for x in colour):
+            raise TypeError("Each colour value must be a float.")
+
+        # Validate scale
+        if not isinstance(scale, float):
+            raise TypeError("Scale must be a float.")
+
+        # Validate position
+        if not isinstance(position, tuple):
+            raise TypeError("Position must be a tuple.")
+        if len(position) != 3:
+            raise ValueError("Position must be a tuple of three values.")
+        if not all(isinstance(x, (int, float)) for x in position):
+            raise TypeError("Each position value must be a float or int.")
+
+        reader = vtk.vtkSTLReader()
+        filepath = os.path.join(
+            os.path.dirname(__file__), "..", "Tabs", "Visualisers", filename
```

Comment:
Modern way of doing this is with `pathlib.Path`


```suggestion
        filepath = Path(__file__).parent / "Tabs" / "Visualisers" / filename
```
Though likely the part up to (and including) "Visualisers" should be a top-level constant.